crates/polars-plan/src/dsl/mod.rs

```diff
@@ -1751,6 +1751,16 @@ pub fn count() -> Expr {
     Expr::Count
 }
 
+/// Cumulative count expression.
+#[cfg(feature = "range")]
+pub fn cum_count(reverse: bool) -> Expr {
+    let mut expr = self::int_range(lit(0), count(), 1, IDX_DTYPE);
```

Comment:
`lit(0 as IdxSize)`. Saves a cast in the optimizer.

As an aside, why `self::`? Isn't it already in scope?

Reply:
The `self` was a brainfart - completely unnecessary.